src/Catty/Extension&Delegate&Protocol/Extensions/UIColorExtension.swift

```diff
@@ -36,6 +36,19 @@ extension UIColor {
     /*
      * from: https://stackoverflow.com/a/24263296
      */
+
+    var redComponent: Int {
+        Int(self.cgColor.components![0] * 255)
```

Comment:
Please avoid force unwrapping in production code and add a test for each component (`CattyTests/Extensions/UIColorEXtensionTests`).